manifests/clusterapi-controller.yaml

```diff
@@ -37,7 +37,11 @@ spec:
         operator: Exists
       containers:
       - name: controller-manager
+      {{- if .AWS }}
+        image: openshift/origin-machine-controller:v4.0.0
```

Comment:
Assuming we already have the `openshift/origin-machine-controller:v4.0.0` image available.

Reply:
https://hub.docker.com/r/openshift/origin-machine-controller/